src/widgets/note_book.rs

```diff
@@ -96,9 +96,12 @@ impl NoteBook {
         }
     }
 
-    pub fn get_current_page(&self) -> i32 {
-        unsafe {
+    pub fn get_current_page(&self) -> Option<u32> {
+        match unsafe {
             ffi::gtk_notebook_get_current_page(GTK_NOTEBOOK(self.pointer))
```

Comment:
Oh.. I'd like that.
I followed the same style/pattern as the other trait methods - should I change those too?